src/xdocs/checks/naming/classtypeparametername.xml

```diff
@@ -87,11 +91,13 @@ class MyClass5&lt;RequestT&gt; {} // violation
         </source>
         <p id="Example3-code">Example:</p>
         <source>
-class MyClass1&lt;T&gt; {}        // violation
-class MyClass2&lt;t&gt; {}        // violation
-class MyClass3&lt;abc&gt; {}      // violation
-class MyClass4&lt;LISTENER&gt; {} // violation
-class MyClass5&lt;RequestT&gt; {} // OK
+class Example3 {
+  class MyClass1&lt;T&gt; {}
```

Comment:
Why isn't this a violation anymore?

Reply:
It was mistake of original doc.
Previously it was just text with any content, now it actual behavior.

Regexp allow just `T`.